examples/waspello/src/cards/MainPage.jsx

```diff
@@ -1,18 +1,18 @@
-import React, { useState, useRef, useContext } from 'react'
```

Comment:
Can I ask you to revert the auto formatting in this file so the change is minimal?

Or reformat with a `.prettierrc` file. If you want, you can add a `.prettierrc` file to the root with this content:
```
{
  "semi": false,
  "singleQuote": true
}
```

Reply:
Thank you for your comment, I've reverted the changes due to auto formatting.